core/src/main/java/org/apache/accumulo/core/client/mock/MockConfiguration.java

```diff
@@ -21,13 +21,16 @@
 
 import org.apache.accumulo.core.conf.AccumuloConfiguration;
 import org.apache.accumulo.core.conf.Property;
+import org.gaul.modernizer_maven_annotations.SuppressModernizer;
 
 import com.google.common.base.Predicate;
 
 /**
  * @deprecated since 1.8.0; use MiniAccumuloCluster or a standard mock framework instead.
  */
 @Deprecated
+@SuppressModernizer
+@SuppressWarnings("overloads")
```

Comment:
I found `javac -X` shows a list of valid ones for use here. I've never seen this one before, but it's in the valid list! :smiley_cat: